docs/src/components/Snippets/glossary_snippets.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+
+export const Glossary = {
+  Tools: {
+    aztec: () => (
+      <p>
+        The CLI tool (with an extensive set of parameters) that enables users to
+        perform a wide range of tasks. It can: run a node, run a sandbox,
+        execute tests, generate contract interfaces for javascript...
+      </p>
+    ),
+    aztec_nargo: () => (
+      <p>
+        The command line tool used to compile Aztec contracts. It gets its name
+        from the Noir compiler,
+        <code>nargo</code>. <code>aztec-nargo</code> is really just a specific
+        version of <code>nargo</code>.
+      </p>
+    ),
+    aztec_up: () => (
+      <p>
+        This tool updates the local aztec executables to the latest version
+        (default behaviour) or to a specified version.
+      </p>
+    ),
+    aztec_wallet: () => (
+      <p>
+        The Aztec Wallet is a CLI wallet, <code>aztec-wallet</code>, that allows
+        a user to manage accounts and interact with an Aztec network. It
+        includes a PXE.
+      </p>
+    ),
+  },
+
+  Libs: {
+    aztec_nr: () => (
+      <p>
+        A <a href="https://noir-lang.org">Noir</a> framework for writing smart
+        contracts on Aztec.
+      </p>
+    ),
+    noir_contracts: () => (
+      <p>
+        Programs that run on the Aztec network, written in the Noir programming
+        language. They may optionally include private state and private
+        functions.
+      </p>
+    ),
+    aztec_js: () => (
+      <p>
+        A{" "}
+        <a href="https://www.npmjs.com/package/@aztec/aztec.js">Node package</a>{" "}
+        to help make Aztec dApps.
+      </p>
+    ),
+  },
+
+  Noir: () => (
+    <p>
+      Noir is a Domain Specific Language for SNARK proving systems. It is used
+      for writing smart contracts in Aztec because private functions on Aztec
+      are implemented as SNARKs to support privacy-preserving operations.
+    </p>
+  ),
+
+  PXE: () => (
+    <p>
+      A client-side key manager, private contract storage, and Private eXecution
+      Environment for private transactions. A PXE is a core part of an Aztec
+      wallet and Sandbox, but can be decoupled and run independently.
```

Comment:
Wdyt about linking to a page with more info here as well?

Reply:
Added link beneath snippet in glossary.md